torchtune/_cli/download.py

```diff
@@ -67,6 +68,18 @@ def _add_arguments(self) -> None:
             default="./model",
             help="Directory in which to save the model.",
         )
+        self._parser.add_argument(
+            "--output-dir-use-symlinks",
+            type=str,
+            required=False,
+            default="False",
```

Comment:
Can we default this to 'auto' to mimic the behavior from before?

Reply:
Sure